BALSAMIC/commands/options.py

```diff
@@ -98,11 +312,94 @@
     help="Instruct Snakemake to not output any progress or rule information",
 )
 
-OPTION_CACHE_VERSION = click.option(
-    "--cache-version",
+OPTION_RULES_TO_DELIVER = click.option(
+    "-r",
+    "--rules-to-deliver",
+    multiple=True,
+    type=click.Choice(DELIVERY_RULES),
+    help="Specify the rules to deliver. The delivery mode selected via the --delivery-mode option.",
+)
+
+OPTION_RUN_ANALYSIS = click.option(
+    "-r",
+    "--run-analysis",
     show_default=True,
-    default=balsamic_version,
+    default=False,
+    is_flag=True,
+    help="Flag to run the actual analysis",
+)
+
+OPTION_RUN_MODE = click.option(
+    "--run-mode",
+    show_default=True,
+    default=RunMode.CLUSTER,
+    type=click.Choice(RUN_MODES),
+    help="Run mode to execute Balsamic workflows",
+)
+
+OPTION_SAMPLE_CONFIG = click.option(
+    "-s",
+    "--sample-config",
+    required=True,
+    type=click.Path(),
+    help="Sample configuration file",
+)
+
+OPTION_SHOW_ONLY_MISSING_FILES = click.option(
+    "-m",
+    "--show-only-missing",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Only show missing analysis files.",
+)
+
+OPTION_SNAKEFILE = click.option(
+    "-S",
+    "--snakefile",
+    type=click.Path(),
+    help="Custom Snakefile for internal testing",
+)
+
+OPTION_SNAKEMAKE_OPT = click.option(
+    "--snakemake-opt",
+    multiple=True,
+    help="Options to be passed to Snakemake",
+)
+
+OPTION_SWEGEN_SNV = click.option(
+    "--swegen-snv",
+    type=click.Path(exists=True, resolve_path=True),
+    required=False,
+    help="VCF path of Swegen SNV frequency database (WGS analysis workflow)",
```

Comment:
I think Swegen is used in TGA as well 🤔  🤔 
```suggestion
    help="VCF path of Swegen SNV frequency database",
```